### PR TITLE
Allow intake users to edit issues from the case details screen

### PR DESCRIPTION
**Pull request by hschallhorn**
Resolves board request from demo

### Description
Allows intake users to see the edit issues link on the case details page. Also removes references to feature toggle that have been enabled for all users for 9 months.

### Testing!
1. Log in as an intake user BVAISHAW and go to an ama case
2. Ensure you can see the "Correct issues" link in case details

### UI Changes
Screen Shot 2020-05-08 at 2 23 48 PM

### CODE REVIEW

File: app/policies/appeal_request_issues_policy.rb [hunk 1]
```diff
@@ -16,19 +16,19 @@ def editable?
...
+  def current_user_can_edit_issues?
+    user.can_edit_issues?
   end
 
   def case_is_not_in_active_review?
     Task.where(appeal: appeal, type: "AttorneyTask").empty?
   end
 
   def hearing_is_assigned_to_judge_user
-    FeatureToggle.enabled?(:allow_judge_edit_issues) && appeal.hearings.last&.judge == user
```
Comment (hschallhorn) [inline app/policies/appeal_request_issues_policy.rb:31]: Feature toggle has been enabled for all for [9 months](https://github.com/department-of-veterans-affairs/appeals-deployment/blob/74ef2bf3f5f0ddfa7bfd33a28a92696dec52b11c/ansible/vars/features-config.yml#L74)

File: spec/feature/queue/case_details/edit_issues_spec.rb [hunk 2]
```diff
@@ -1,48 +1,53 @@
 # frozen_string_literal: true
 
 RSpec.feature "Edit Issues", :all_dbs do
-  before do
-    FeatureToggle.enable!(:allow_judge_edit_issues)
-  end
+  shared_examples "can NOT view the 'Correct Issues' link" do
```
Comment (leikkisa) [inline spec/feature/queue/case_details/edit_issues_spec.rb:4]: I need to learn more about this test set up structure but it's pretty cool!

File: app/policies/appeal_request_issues_policy.rb [hunk 3]
```diff
@@ -16,19 +16,19 @@ def editable?
...
 
   def editable_by_case_review_team_member?
-    current_user_is_case_review_team_member? && case_is_not_in_active_review?
+    current_user_can_edit_issues? && case_is_not_in_active_review?
   end
 
-  def current_user_is_case_review_team_member?
-    CaseReview.singleton.user_has_access?(user)
+  def current_user_can_edit_issues?
+    user&.can_edit_issues?
```
Comment (hschallhorn) [inline app/policies/appeal_request_issues_policy.rb:23]: AppealRequestIssuesPolicy is only used in the [appeals serializer](https://github.com/department-of-veterans-affairs/caseflow/blob/49353b6af91c3da50a9241c6fdf8be9116a57879/app/models/serializers/work_queue/appeal_serializer.rb#L41). [Reader does not pass the current user to this serializer](https://github.com/department-of-veterans-affairs/caseflow/blob/3e0639a69202486a15e4b2b8bb763754c14b236e/app/controllers/reader/appeal_controller.rb#L29), so we may not always have a user to check.

File: spec/feature/queue/case_details/edit_issues_spec.rb (hunk 2 above)
Comment (hschallhorn) [reply spec/feature/queue/case_details/edit_issues_spec.rb:4]: It's helped a lot with code climate code duplication warnings in tests!

**Pull request merged**